simd.dd

```diff
@@ -92,8 +100,8 @@ $(H3 Conversions)
 	to their vector equivalents:)
 
 ---
-int4 v = 7;
```

Comment:
It's not that int4 v = 7; doesn't work, it's the next line that doesn't work. v = 3 * v; does not work because int4 does not have multiplication defined. See the table at the bottom. That's why I made the change. v = 3 + v; does work. But I changed it to a float so that I kept it as multiplication. I watched Manu's talk from DConf 2013 last night and he actually recommended not even doing this kind of thing anyway, so I'm tempted to add that in as well.